### DISCUSSION TITLE
使用官方提供的图片进行表格识别出现误差

### DISCUSSION THREAD

**Discussion by luoyq6** [PaddlePaddle/PaddleOCR#12931] [Q&A]
使用命令 python .\table\predict_table.py --image_dir=D:/project_code/PaddleOCR/test_img/table.jpg --output=../output/table_1 出现误差，是需要修改其他参数吗？
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/110521846/99a96ed1-7dfe-4aed-b6cd-c6f5afc7f59f)

![image](https://github.com/PaddlePaddle/PaddleOCR/assets/110521846/bd473259-867e-46e8-aab1-a79df9bdf982)

**Comment by wzg722** [comment]
你裁剪一下表格出来，然后识别
